content/post/stash-v2022.06.27/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing Stash v2022.06.27
+date: 2022-06-27
+weight: 10
+authors:
+  - Hossain Mahmud
+tags:
+  - kubernetes
+  - stash
+  - backup
+  - restore
+---
+
+We are very excited to announce Stash `v2022.06.27`. In this release, we have fixed some bugs. You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2022.06.27/README.md).
+
+We are going to highlight the changes in this post.
+
+## Bug Fixes
+
+In this release we have fixed a bug that was causing the license check to fail in the backup and restore job of Postgres ([#1077](https://github.com/stashed/postgres/pull/1077)) and MongoDB ([#1593](https://github.com/stashed/mongodb/pull/1593)) datbases.  
```

Comment:
You don't need to separate the sections just for a bug fix. Just do something like this:

```markdown
We are announcing Stash `v2022.06.27` which include a critical bug fix for PostgreSQL and MongoDB addon. This release fixes a bug that was causing the license checker to fail in the backup and restore jobs for PostgreSQL ([#1077](https://github.com/stashed/postgres/pull/1077)) and MongoDB ([#1593](https://github.com/stashed/mongodb/pull/1593)) databases.

You can find the complete changelog [here](https://github.com/stashed/CHANGELOG/blob/master/releases/v2022.06.27/README.md).
```